tests/end2end/data/userscripts/open_current_url.bat

```diff
@@ -0,0 +1 @@
+echo open -t %QUTE_URL% >> "%QUTE_FIFO%"
```

Comment:
Seems like I should finally set up an editor on Windows correctly if I continue to do this Windows stuff :laughing: